dev/NavigationView/NavigationView.cpp

```diff
@@ -501,28 +874,15 @@ void NavigationView::OnLayoutUpdated(const winrt::IInspectable& sender, const wi
     }
```

Comment:
I'm not entirely sure why we need to do hookup to LayoutUpdated. I was think we could do all the layout synchronously and not have to do multiple passes.

Reply:
Removed the logic that triggered another layout pass